### DISCUSSION TITLE
✅ Why can't it be updated?

### DISCUSSION THREAD

**Discussion by Limiyali** [isontheline/pro.webssh.net#227] [Q&A] [answered]
When I update, it shows that the developer has removed this app from the app store

**Comment by isontheline** [comment]
Hello @Limiyali 👋

On which App Store country?

**Reply by isontheline** [reply]
Could you provide :
- screenshot of the error
- iOS version
- country of your iDevice

**Reply by Limiyali** [reply]
I switched the appid to usa, then uninstalled and reinstalled, the problem was solved.
My original id country is China
（now，I find This app has been deleted from the AppStore in China）

**Reply by isontheline** [reply]
Yes WebSSH has been retired from China App Store due to the lack of proper VPN license.

I will work in the next months to comply with all China Territory Legal Rules

Have a nice day ☀️

**Comment by mbierman** [comment, marked as answer]
It is in the US store.

**Reply by isontheline** [reply]
Please provide a screenshot

**Reply by isontheline** [reply]
Was it a question or an affirmation @mbierman ?

Thank you

**Reply by mbierman** [reply]
Just affirmation.sorry for the typos (now fixed).

**Reply by Limiyali** [reply]
thank you

**Reply by isontheline** [reply]
Thank you to you @mbierman @Limiyali 👍
